.rubocop.yml

```diff
@@ -508,7 +508,7 @@ Metrics/LineLength:
   Description: Limit lines to 100 characters.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#80-character-limits
   Enabled: true
-  Max: 100
+  Max: 80
```

Comment:
After meeting with you yesterday, perhaps it is your Rubocop in Ruby-mine which is running with a different config? 
Check if you can use the .rubocop file.